examples/LFPrecording/cell_lfp.py

```diff
@@ -34,11 +34,18 @@
 simConfig.recordLFP = [[x, y, 35] for y in range(280, 1000, 150) for x in [30, 90]]
 
 simConfig.analysis['plotTraces'] = {'include': [('E',0)], 'oneFigPer':'cell', 'overlay': True, 'figSize': (5,3),'saveFig': True}      # Plot recorded traces for this list of cells
-simConfig.analysis['plotLFP'] = {'includeAxon': False, 'plots': ['timeSeries',  'locations'], 'figSize': (5,9), 'saveFig': True} 
+simConfig.analysis['plotLFP'] = {'includeAxon': False, 'plots': ['timeSeries',  'locations'], 'figSize': (5,9), 'saveFig': True}
+simConfig.analysis['getCSD'] = {'timeRange': [10,45],'spacing_um': 150, 'vaknin': True}
+
+#sim.analysis.getCSD(...args...)
+#simConfig.analysis['plotCSD'] = {}
+
 
 # Create network and run simulation
 sim.createSimulateAnalyze(netParams = netParams, simConfig = simConfig)    
+sim.analysis.plotCSD()
```

Comment:
For whatever reason, this file fails when run on Python 2.  I'd guess it's because plotCSD fails, but this needs to be fixed.